jekyll-theme-feeling-responsive.gemspec

```diff
@@ -0,0 +1,23 @@
+Gem::Specification.new do |s|
+  s.name = 'jekyll-theme-feeling-responsive'
+  s.version = '1.0.2'
```

Comment:
1.9.1 https://phlow.github.io/feeling-responsive/changelog/

Reply:
Hi! Thank you for the review.  Are you suggesting an update to the change log?  I searched the referenced link for mention of this feature.  Has it already been done?

The change log is for the `gh-pages` branch.  It doesn't exist on the `bare-bones-version` branch.  This change is to the `bare-bones-version` branch.  Do you think the change belongs on the `gh-pages` branch?

Reply:
Ah, then probably it's correct, @Phlow can always update.
I meant to not diverge release and gem version.